packages/backend/src/runtime/index.ts

```diff
@@ -12,9 +12,9 @@
  * TODO: Support TS runtime modules
  */
 
-// @ts-expect-error
+// @ts-ignore - These are package subpaths
```

Comment:
When running the tests, this threw an error because for the `tsconfig.test.json` this didn't seem to fail. It's one of the quirks about our `tsconfig` setup so let's disregard it for this PR

Reply:
I think we need to update the tests in `backend` package to build with `tsup` instead of `tsc` and `tsconfig.test.json`.  (that change is not part of the current PR)